src/utilities/effects.ts

```diff
@@ -3717,7 +3718,13 @@ const effects = {
 
       const { update_activity_presets_by_pk } = await reqHasura<ActivityPreset>(
         gql.UPDATE_ACTIVITY_PRESET,
-        { ...updatedActivityPreset, id },
+        {
+          activityPresetSetInput: {
+            ...updatedActivityPreset,
+            id: undefined,
```

Comment:
is setting the `id` of the preset explicitly to `undefined` necessary?

Reply:
the id is not allowed here, but I updated to use the standard style here